src/components/Dropdown/Dropdown.md

```diff
@@ -38,7 +38,7 @@ By default the dropdown menu will not close when an item within the dropdown is
 
 ```jsx
 <Dropdown label="Toggle On Click" toggleDropdownOnClickInside>
-    <Button variant="plain">Item one</Button>
-    <Button variant="plain">Item two</Button>
+    <Button onClick={() => alert('Clicked Item one')}>Item one</Button>
+    <Button onClick={() => alert('Clicked Item one')}>Item two</Button>
```

Comment:
Did you mean for this alert to read `'Clicked Item two'`?

Reply:
Yes, good catch!